Patches/Core/Stats/Stats.xml

```diff
@@ -56,28 +56,40 @@
 	<!-- Shooting Accuracy -->
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>*/StatDef[defName="ShootingAccuracy"]/label</xpath>
+		<xpath>*/StatDef[defName="ShootingAccuracyPawn"]/label</xpath>
 		<value>
 			<label>weapon handling</label>
 		</value>
 	</Operation>
 
+	<Operation Class="PatchOperationSequence">
```

Comment:
Why are we running the same operation twice?

Reply:
First operation is PatchOperationTest and second is PatchOperationRemove

Reply:
Ah didn't spot that. Only issue then is this PR should go into the development branch.

Reply:
Alright, corrected